src/coreComponents/physicsSolvers/SolverBase.hpp

```diff
@@ -53,6 +53,11 @@ class SolverBase : public ExecutableGroup
 
   static string catalogName() { return "SolverBase"; }
```

Comment:
@rrsettgast should we remove all those `base` catalogNames?

Reply:
yes...but we can do that in its own PR. We can also do some of what we talked about yesterday.